HRTool/src/main/java/app/generator/ChartGenerator.java

```diff
@@ -135,12 +138,25 @@ public static void generateTechnology(List<ProjectInfo> projInfos, TagCloudEnum
 		}
 		Document doc = new Document();
 		PdfWriter writer = null;
-		if (tce.equals(TagCloudEnum.Technologie)) {
-			writer = PdfWriter.getInstance(doc, new FileOutputStream(new File("./technology-chart.pdf")));
-		}
-		else {
-			writer = PdfWriter.getInstance(doc, new FileOutputStream(new File("./database-chart.pdf")));
+		String fileName = null;
+		File pc = null;
+		switch (tce){
+			case Technologie:
+				currentDate = new Date();
+				fileName = "./technology-chart_" + sdf.format(currentDate) + ".pdf";

```

Comment:
Constant strings should be extracted. Create a method for creating file names based on a Technology. Also note how "Technology" is spelled.